convert/terraform/outputs.tf

```diff
@@ -1,49 +1,4 @@
 output "domino_eks" {
```

Comment:
is this output used anymore?

Reply:
Was debating whether I wanted to keep it or not. The other outputs are specifically for the deployer, but this here (alone, without those outputs) is what I would probably do if I were making something that was only ever used without the deployer. As this is used with both, but has no obligation to track the deployer outputs as-is (since the deployer will now overwrite main.tf.json), there's maybe still both user-friendliness/completeness arguments for this for any usage of this that doesn't coincide with the deployer.